spec/swagger.yaml

```diff
@@ -8,8 +8,6 @@ info:
     between ONE and your existing ecosystem.
   version: 1.0.0
   title: Datastores API documentation
-  x-logo:
```

Comment:
This was also probably auto-updated and may need to be reverted.  @andreipop please advise.